src/table/virtualized-table/Body.tsx

```diff
@@ -102,6 +98,14 @@ const Body = forwardRef<BodyRef, BodyProps>((props, ref) => {
     gridRef.current.resetAfterIndices({ columnIndex: 0, rowIndex: 0, shouldForceUpdate: true });
   }, [layout, pageInfo.page, gridRef, columnWidth, rowMeta, theme.name()]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  useLayoutEffect(() => {
+    gridRef.current?.scrollTo({ scrollTop: 0 });
+  }, [columnWidth, rowCount, columns.length, pageInfo]);
+
+  useOnPropsChange(() => {
+    updateCellHeight(rowsInPage);
```

Comment:
This is what triggers the cells to be re-calculated and re-rendered when the user resizes a column